templates/kubernetes/teraform/README.md

```diff
@@ -0,0 +1,29 @@
+# EKS Terraform 
+
+AWS Resources created: 
+
+- EKS Cluster: AWS managed Kubernetes cluster of master servers
+- AutoScaling Group containing 2 m4.large instances based on the latest EKS Amazon Linux 2 AMI: Operator managed Kubernetes worker nodes for running Kubernetes service deployments
+- Associated VPC, Internet Gateway, Security Groups, and Subnets: Operator managed networking resources for the EKS Cluster and worker node instances
+- Associated IAM Roles and Policies: Operator managed access resources for EKS and worker node instances
+
+## Pre-requisites
+
+- Setup the [AWS credentials](https://www.terraform.io/docs/providers/aws/index.html#environment-variables) for terraform 
+
+## Spin up cluster
+
+```shell
+
+terraform plan 
```

Comment:
Shouldn't we roll all this stuff into the application to run?

Reply:
By application run, do you mean the result of `commit0 generate -l kubernetes`?

Reply:
In that event I should also create an s3 bucket and save the `tf.state` there.

Reply:
Maybe this is fine for now, but I feel like we need a new command like `commit0 run` that actually runs everything. Because generate may create a whole bunch of stuff that needs to get run by various applications. I'll mention it in Slack

Reply:
As per slack convo, added `deploy` flag for the `commit0.yaml kuberntes config`, if it's set to true, will run `terraform init & terraform plan`